dpcpp/base/helper.hpp

```diff
@@ -62,10 +62,12 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
                InferredArgs... args)                                      \
     {                                                                     \
         queue->submit([&](sycl::handler& cgh) {                           \
-            cgh.parallel_for(sycl_nd_range(grid, block),                  \
-                             [=](sycl::nd_item<3> item_ct1) {             \
-                                 kernel_(args..., item_ct1);              \
-                             });                                          \
+            cgh.parallel_for(                                             \
+                sycl_nd_range(grid, block), [=                            \
+            ](sycl::nd_item<3> item_ct1) [[sycl::reqd_sub_group_size(     \
+                                                config::warp_size)]] {    \
```

Comment:
should we leave as the default one not config::warp_size?

Reply:
I would leave it as warp_size, since that's als what we are using inside a few of these kernels (mostly SpMV)